docs/expectation_gallery/README.md

```diff
@@ -23,14 +22,14 @@ This will happen if the `CamelCaseName` (Expectation class name) and `snake_case
 
 ### Newly merged Expectation is missing from the staging Expectation Gallery
 
-> <https://staging-great-expectations.netlify.app/expectations>
+> &lt;https://staging-great-expectations.netlify.app/expectations&gt;
```

Comment:
This seems like weird formatting. Should it just be the bare url instead? This is also outside of the docusaurus directory, so we technicallydon't need to touch it.

Reply:
@tyler-hoffman it can be a bare URL, or it can include the chevrons. I left them in. I don't feel strongly one way or the other.